static/scss/channel.scss

```diff
@@ -52,6 +52,8 @@
   width: 100%;
 
   .channel-page-header {
+    margin-bottom: 20px;
```

Comment:
I think this should be a `margin-top` on `.widget-list` instead, because it's pushing the post list column down too which looks worse on mobile views. Either that or we can just punt further design tweak to another PR, your choice.

Reply:
My intention was to match the existing behavior on the home screen which has a 20px margin at the bottom of the `home-callout` `Card`: https://github.com/mitodl/open-discussions/blob/master/static/scss/card.scss#L5

I'd like to keep it as is for now and tweak that later so we can do the corresponding home page change at the same time